### ISSUE TITLE
Please help to update module path

### ISSUE THREAD

**Issue by TommyLeng** [musclechen/disgo#6]
![image](https://user-images.githubusercontent.com/95701513/184563770-0bf3b5f8-c543-40e1-be16-7e64fdd59767.png)

It is using cyfckl2012

**Comment by musclechen** [comment]
I have completed the update, thanks.

**Comment by TommyLeng** [comment]
I want to ask... why the publish channel name is different from the subscribe channel name.

luaRelease publish is using lockZSetName

But subscribe  function is using lockPublishName...

But it seems working...

**Comment by musclechen** [comment]
Because I wrote this code for a long time, I don't really remember how it was designed at the time. Maybe this is a bug, if you have time you can try changing lockZSetName to lockPublishName and see if it works. 

If it can be determined to be a bug, you are welcome to submit a PR after modification and testing. I have no work in the golang, so I don't have extra time to verify this problem. 

Thanks again.

**Comment by TommyLeng** [comment]
Thx for your reply. I will check it.

I am originally using the lib https://github.com/bsm/redislock.

But I want the lock with FIFO feature. 

I guess now the code works because the cas function will try to help lock too if subscribe function is failed.
I will check it later.

**Comment by musclechen** [comment]
If you want an absolute FIFO lock, then disgo may not be suitable, because CAS and subscribe cannot guarantee absolute FIFO. The disgo uses zset as a queue before CAS, which guarantees FIFO to a certain extent. In the case of high concurrency, after the original lock is released, it is the turn of the X process to lock, but the CPU happens to be time sliced, the X process is in a dormant state and does not wake up in time, a new process comes in and grabs the lock, then the X process will still subscribed.

In other words, if your requirement is 100% guaranteed FIFO, you may need to make further modifications.
